src/codeflare_sdk/__init__.py

```diff
@@ -19,3 +19,13 @@
 from .job import RayJobClient
 
 from .utils import generate_cert
+
+from importlib.metadata import version, PackageNotFoundError
+
+try:
+    __version__ = version(
+        "codeflare-sdk"
+    )  # Update with latest version with each release
```

Comment:
Just one question, wondering what do you mean by this comment?

Reply:
I meant to say that It should get updated with every release. We can remove it if its not required or makes it confusing.

Reply:
Ohh I understand, that makes sense. I saw the comment as an "action item" so got confused. I'd suggest to remove the comment. Great work, thanks!